server/spec/mutations/stacks/deploy_spec.rb

```diff
@@ -39,47 +44,16 @@
         ]
       )
       stack.grid_services.destroy_all
+      expect(worker_async).to receive(:perform).once
       expect {
         described_class.run(stack: stack)
       }.to change{ stack.grid_services.count }.by(2)
     end
 
-    it 'removes services that are removed from a stack' do
-      Stacks::Update.run(
-        stack_instance: stack,
-        name: 'stack',
-        stack: 'foo/bar',
-        version: '0.1.0',
-        registry: 'file://',
-        source: '...',
-        services: [
-          {name: 'redis', image: 'redis:2.8', stateful: true },
-          {name: 'nginx', image: 'nginx:latest', stateful: false }
-        ]
-      )
-      Stacks::Update.run(
-        stack_instance: stack,
-        name: 'stack',
-        stack: 'foo/bar',
-        version: '0.1.0',
-        registry: 'file://',
-        source: '...',
-        services: [
-          {name: 'nginx', image: 'nginx:latest', stateful: false }
-        ]
-      )
-      worker = Celluloid::Actor[:stack_deploy_worker]
-      redis = stack.grid_services.find_by(name: 'redis')
-      mutation = described_class.new(stack: stack)
-      mutation.run
-      sleep 0.01 until worker.mailbox.size == 0
```

Comment:
This spec was flaky before... the actor can have tasks running even if the mailbox is empty